includes/RestApi/PluginsController.php

```diff
@@ -322,6 +322,17 @@ public function set_site_features( \WP_REST_Request $request ) {
 
 		foreach ( $plugins as $plugin => $decision ) {
 			if ( $decision ) {
+				// If the Plugin exists and is activated
+				if ( PluginInstaller::exists( $plugin, $decision ) ) {
+					continue;
+				}
+
+				// If the Plugin is already in queue
+				$position_in_queue = PluginInstallTaskManager::status( $plugin );
+				if ( false !== $position_in_queue ) {
+					continue;
+				}
```

Comment:
We don't need to do this, the `add_to_queue` function takes care of preventing duplication in the queue.

Reply:
Yeah Sure, Done.